### PR TITLE
App 284 add markdown support for messages

### PR DESCRIPTION
**Pull request by tarikstafford**

### CODE REVIEW

File: Projects/hCoreUI/Sources/MarkdownText.swift [hunk 1]
```diff
@@ -53,6 +54,8 @@ extension MarkdownText: Viewable {
             }
         }
 
+        let tapGesture = UITapGestureRecognizer()
```
Comment (sampettersson) [inline Projects/hCoreUI/Sources/MarkdownText.swift:57]:
```suggestion
```

File: Projects/hCoreUI/Sources/MarkdownText.swift [hunk 2]
```diff
@@ -41,6 +41,7 @@ extension MarkdownText: Viewable {
         markdownText.numberOfLines = 0
         markdownText.lineBreakMode = .byWordWrapping
         markdownText.baselineAdjustment = .none
+        markdownText.isUserInteractionEnabled = true
```
Comment (sampettersson) [inline Projects/hCoreUI/Sources/MarkdownText.swift:44]:
```suggestion
```

**Review by sampettersson — CHANGES_REQUESTED** [review]
Good find on using a UITextView instead ⛱️

File: Projects/hCoreUI/Sources/MarkdownText.swift (hunk 2 above)
Comment (tarikstafford) [reply Projects/hCoreUI/Sources/MarkdownText.swift:44]: I think you need this for links to be tappable.

**Review by sampettersson — APPROVED** [review]

**Pull request merged**